src/Bridge/Symfony/DependencyInjection/FidryAliceDataFixturesExtension.php

```diff
@@ -44,12 +41,12 @@ public function load(array $configs, ContainerBuilder $container)
         $processedConfiguration = $this->processConfiguration($configuration, $configs);
         $bundles = array_flip($container->getParameter('kernel.bundles'));
 
-        if (false === array_key_exists(self::NELMIO_ALICE_BUNDLE_CLASS, $bundles)) {
+        if (false === array_key_exists(NelmioAliceBundle::class, $bundles)) {
```

Comment:
Hm maybe the problem I stumbled on was a different one then. You can leave this change in the PR it's not much and well scoped. Cool to simplify a thing or two :)